src/js/control-bar/text-track-controls/caption-settings-menu-item.js

```diff
@@ -54,6 +54,12 @@ class CaptionSettingsMenuItem extends TextTrackMenuItem {
   handleClick(event) {
     this.player().getChild('textTrackSettings').open();
   }
+
+  // CaptionSettingsMenuItem does not need to handle track list change events because
+  // it has no concept of 'selected'
+  handleTracksChange(event) {
```

Comment:
this hopefully shouldn't be necessary because we're setting `selectable` to false and `selected` already does a check for it.

Reply:
Ah yep. Overlooked the fact that the `selectable` check makes it a non-issue